solutions/observability/apps/apm-server-api.md

```diff
@@ -10,7 +10,6 @@ The APM Server exposes endpoints for:
 * [APM Server information API](apm-server-information-api.md)
 * [Elastic APM events intake API](elastic-apm-events-intake-api.md)
 * [Elastic APM agent configuration API](elastic-apm-agent-configuration-api.md)
-* [OpenTelemetry intake API](opentelemetry-intake-api.md)
```

Comment:
I'm not sure about whether or not we should remove this. We'll have to check with @akhileshpok.